modules/creating-a-machine-pool-cli-capres.adoc

```diff
@@ -6,31 +6,27 @@
 [id="creating_machine_pools_cli_capres_{context}"]
 = Creating a machine pool with Capacity Reservations using the {rosa-cli}
 
+[role="_abstract"]
 You can create a new machine pool with Capacity Reservations by using the {rosa-cli-first}. Both On-Demand Capacity Reservations and Capacity Blocks for ML are supported.
 
-[NOTE]
-====
-Currently, upgrading machine pools and enabling autoscaling is not suppored on machine pools with Capacity Reservations.
-====
-
```

Comment:
Sadly, this will be the case still for 1.2.58 - in 1.2.59 we will be removing the gating which blocks `autoscaling + cap rez`

(also, there is a typo in `supported` if we put this back in)